src/tools/tidy/src/extdeps.rs

```diff
@@ -25,6 +25,12 @@ pub fn check(root: &Path, bad: &mut bool) {
         // Extract source value.
         let source = line.split_once('=').unwrap().1.trim();
 
+        // rustc only permits crates from crates.io so for now libgc_internal is
+        // hardcoded to pass the tidy checker.
+        if source.contains("softdevteam/libgc_internal") {
```

Comment:
Can we make this `ends_with` or similar, just so that it's less likely to trigger incorrectly?

Reply:
Unfortunately not. The actual sources list looks something like this:`<metadata>softdevteam/libgc_internal<hash_which_changes_when_package_updates>`

Reply:
Ah, OK.